notebooks/learning-to-rank/01-learning-to-rank.ipynb

```diff
@@ -0,0 +1,1513 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from xgboost import plot_importance\n",
+    "\n",
+    "plot_importance(ranker, importance_type=\"weight\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing the model to Elasticsearch\n",
+    "\n",
+    "Once the model is trained you will be able to use Eland to send it to Elasticsearch.\n",
+    "\n",
+    "Please note that the `MLModel.import_ltr_model` method contains the `LTRModelConfig` object in order to associate the feature extraction with the model.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 144,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "zAMwvqYlq9py",
+    "outputId": "c0f60ce3-fb07-47a5-9e37-fccbd1f30bcc"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/afoucret/git/elasticsearch-labs/.venv/lib/python3.9/site-packages/eland/ml/ml_model.py:550: ElasticsearchWarning: The default [remove_binary] value of 'false' is deprecated and will be set to 'true' in a future release. Set [remove_binary] explicitly to 'true' or 'false' to ensure no behavior change.\n",
```

Comment:
do we want to set this parameter explicitly so the warning disappears?

Reply:
This is something to be fixed into Eland and not in the notebook. I will open a PR for it.